platform/build.gradle

```diff
@@ -16,3 +16,9 @@ dependencies {
 moduleJvmArgs {
     opens 'java.base/java.lang'
 }
+/**
+ * Needs to be kept at Java 11 to keep compatibility with <a href="https://github.com/palantir/gradle-jdks/blob/develop/gradle-jdks-setup/build.gradle#L29">gradle-jdks-setup</a>
+ */
```

Comment:
I'd not use javadoc in gradle build scripts and just use normal `//` comments, it doesn't do anything special (like get rendered anywhere), and it's generally easier for people to read non-raw html.